cmd/podman/images/pull.go

```diff
@@ -154,15 +154,22 @@ func imagePull(cmd *cobra.Command, args []string) error {
 	}
 	// Let's do all the remaining Yoga in the API to prevent us from
 	// scattering logic across (too) many parts of the code.
-	pullReport, err := registry.ImageEngine().Pull(registry.GetContext(), args[0], pullOptions.ImagePullOptions)
-	if err != nil {
-		return err
+	pulledImages := make([]string, 0)
+	for _, arg := range args {
+		pullReport, err := registry.ImageEngine().Pull(registry.GetContext(), arg, pullOptions.ImagePullOptions)
+		if err != nil {
+			return err
+		}
+		pulledImages = append(pulledImages, pullReport.Images...)
+		if !pullOptions.Quiet {
+			fmt.Print("\n")
```

Comment:
Is this correct?  If I pull a bunch of images with quiet, doesn't it print the ImageID for each image I pull? Why doesn't that need a \n for more then one?

Reply:
I would expect the output to be
```
$ podman pull ubi8-micro fedora
Trying to pull registry.access.redhat.com/ubi8-micro:latest...
Getting image source signatures
Copying blob c07a48b26be2 skipped: already exists  
Copying blob 4f4fb700ef54 [--------------------------------------] 0.0b / 0.0b
Copying config f390b26f6a done  
Writing manifest to image destination
Storing signatures
f390b26f6a00561dbacfc5991a39a6845ff7968f8d18a319b68cc9c0f97dea73
Trying to pull registry.fedoraproject.org/fedora:latest...
Getting image source signatures
Copying blob ca594071dfff skipped: already exists  
Copying config abec9a7a7d done  
Writing manifest to image destination
Storing signatures
abec9a7a7dc640768e6dc51b6a5728e470411615c62e9ff46215206bde816772
```
Rather then

```
$ podman pull ubi8-micro fedora
Trying to pull registry.access.redhat.com/ubi8-micro:latest...
Getting image source signatures
Copying blob c07a48b26be2 skipped: already exists  
Copying blob 4f4fb700ef54 [--------------------------------------] 0.0b / 0.0b
Copying config f390b26f6a done  
Writing manifest to image destination
Storing signatures
Trying to pull registry.fedoraproject.org/fedora:latest...
Getting image source signatures
Copying blob ca594071dfff skipped: already exists  
Copying config abec9a7a7d done  
Writing manifest to image destination
Storing signatures
Pulled Images:
f390b26f6a00561dbacfc5991a39a6845ff7968f8d18a319b68cc9c0f97dea73
abec9a7a7dc640768e6dc51b6a5728e470411615c62e9ff46215206bde816772
```

Reply:
I don’t feel strongly about this, but the second one seems _much_ easier to copy&paste. (Of course users should not copy&paste non-API command output, but then why are the IDs printed at all?)

Reply:
Because docker printed them, and they make sense when you do it with a -q option on one image.

Reply:
```
❯ docker pull -q busybox
docker.io/library/busybox:latest
```
Docker doesnt print the IDs for -q flag. 
So let me know what the desired behavior is and I will make the changes accordingly

Reply:
I would prefer to print the image name on success, but I think we have set the precedent of printing the ID.  WDYT @vrothberg @mheon ?

Reply:
I think we should do what @rhatdan suggested in https://github.com/containers/podman/pull/10788#discussion_r660486435.